src/components/DeleteComment.jsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from "react";
+import { getCommentsByArticleId, deleteComment } from "../api";
+
+ export function DeleteComment({comment, setComments, setClickDelete}) {
+
+  const [username, setUsername] = useState("");
+  const [password, setPassword] = useState("");
+  const [message, setMessage] = useState("");
+
+  const handleDelete = (e) => {
+    e.preventDefault();
+    if(username !== comment.author) {
+        setMessage("Username invalid. Check spelling and ensure you are only trying to delete your own comment!");
+        return;
+    }
+    if(password !== "password") {
+        setMessage("Incorrect password.");
+        return;
+    }
+        deleteComment(comment.comment_id)
+          .then(() => {
+            setComments((currentcomments) => {
+             return currentcomments.filter((item) => item.comment_id !== comment.comment_id )})
+            setMessage("Comment deleted successfully.");
+            setClickDelete(false)
+          })
+          .catch((error) => {
+            console.log(error)
+            setMessage("Error deleting comment:"+ error);
+          });
+
+  };
+
+  return (
+    <div>
+      <h3>Delete Comment</h3>
+      <form onSubmit={handleDelete}>
+        <div>
+          <label htmlFor="username">Username:</label>
+          <input
+            type="text"
+            value={username}
+            onChange={(e) => setUsername(e.target.value)}
+            required
+          />
+        </div>
+        <div>
+          <label htmlFor="password">Password:</label>
+          <input
+            placeholder="password is password"
+            type="password"
+            value={password}
+            onChange={(e) => setPassword(e.target.value)}
+            required
+          />
```

Comment:
What is the point of a password box when none of the users actually have a password in the db? 😅 

I'd definitely say we can skip this step. It would make more sense if there were passwords in the db, but our backend isn't currently built for that.

Also, for user experience you wouldn't want to force the user to type in their username and password over and over to delete multiple comments as it would get annoying fast!